DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: r2dii.analysis
 Title: Tools to Calculate Climate Targets for Financial
     Portfolios
-Version: 0.0.0.9004
+Version: 0.0.1
```

Comment:
I think this should be 0.0.1.900. We can chat a bit about this. It's something I have never fully grasped but I think this time I'll refine my thoughts with your help, then  capture them in writing -- so I don't stress about this silly detail ever again.